### PR TITLE
Allow Ctrl+Space

### PR DESCRIPTION
**Pull request by alexwforsythe**
I'd like to record tapes that can type Ctrl+Space to trigger a keybinding I have in tmux. I could achieve this by remapping the keybinding temporarily for VHS, but it was nearly as easy to just make this PR instead.

A similar change was made recently in https://github.com/charmbracelet/vhs/commit/b5f8be82aec8d538bd00149985fdb9dfa50e5d4f to allow Ctrl+Shift+Alt+Enter, so I figured this might also be acceptable.

### CODE REVIEW

File: parser_test.go [hunk 1]
```diff
@@ -207,48 +207,73 @@ func TestParseTapeFile(t *testing.T) {
...
+		},
+		{
+			name:     "should parse with multiple modifiers",
+			tape:     "Ctrl+Shift+Space",
+			wantArgs: []string{"Shift", "Space"},
+			wantErr:  false,
+		},
+		{
+			name:     "should parse with multiple modifiers",
+			tape:     "Ctrl+Space+Shift",
```
Comment (maaslalani) [inline parser_test.go:236]: I think this should be an error as Space is a key and should not come before a modifier.

File: parser_test.go [hunk 2]
```diff
@@ -207,48 +207,73 @@ func TestParseTapeFile(t *testing.T) {
...
+			wantArgs: []string{"Space", "Shift"},
+			wantErr:  false,
+		},
+		{
+			name:    "Ctrl+AltRight",
+			wantErr: true,
+		},
+		{
+			name:    "Ctrl+Backspace",
```
Comment (maaslalani) [inline parser_test.go:245]: We should probably add Ctrl+Backspace as valid, but this makes sense for now!

**Review by maaslalani — APPROVED** [review]

File: parser_test.go (hunk 1 above)
Comment (alexwforsythe) [reply parser_test.go]: Done! Added in a new commit

File: parser_test.go (hunk 2 above)
Comment (alexwforsythe) [reply parser_test.go]: Done! Added in a new commit

**Comment by maaslalani** [comment]
Thank so much for the contribution @alexwforsythe!

**Pull request merged**